packages/core/admin/ee/admin/src/pages/SettingsPage/pages/ReviewWorkflows/CreatePage.tsx

```diff
@@ -100,7 +100,7 @@ export const ReviewWorkflowsCreatePage = () => {
         },
       });
 
-      navigate(res.data.id.toString());
+      navigate(`/settings/review-workflows/${res.data.id}`);
```

Comment:
we need to fix our router.

Reply:
I don't see how this could ever have worked even by changing the base path of the router. it would move relative to the current url so add the id after /create anyway 🤔

Reply:
if we set the router up correctly with the right levels of nesting we can do `../${id}` which is just a bit easier to work with in the long run

Reply:
🤔 that looks like a pretty fragile routing to me if you move the creation page anyware all the relative links in that page will break directly. 

It would make sense to have the parent basePath be `settings/review-workflows` so you can navigate to `/id` instead